.github/workflows/cigh.yml

```diff
@@ -0,0 +1,125 @@
+      CC: /usr/bin/clang
+      CXX: /usr/bin/clang++
+    steps:
+      - name: Check out repository ref ${{ github.ref }} of ${{ github.repository }}
+        uses: actions/checkout@v2
+## NINJA
+      #- name: Creating build directory
+        #run: mkdir -p $CI_BUILD_DIR
+## CCACHE
+      - name : Creating build and Ccache directories
+        run: mkdir -p $CI_BUILD_DIR && mkdir -p $CI_CCACHE_DIR && mkdir -p $CCACHE_CONFIGPATH && mkdir -p $CCACHE_DIR
+##
+      - name: Setting cache
+        uses: actions/cache@v2
+        with:
+## NINJA
+          #path: ${{ github.workspace }}/build # !! YAML node anchors not supported by GH !!
+## CCACHE
+          path: ${{ github.workspace }}/ccache # !! YAML node anchors not supported by GH !!
+##
+          key: ${{ runner.os }}-FC-${{ github.ref }}-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-FC-${{ github.ref }}-
+            ${{ runner.os }}-FC-
+      - name: Configuring CMake
+## NINJA
+        #run: cmake -GNinja -B$CI_BUILD_DIR .
+## CCACHE
+        run: cmake -B$CI_BUILD_DIR .
+# Below is minimal build for CI testing purposes
+        #run: >
+          #cmake -B$CI_BUILD_DIR .
+          #-DBUILD_ADDONMGR=OFF
+          #-DBUILD_ARCH=OFF
+          #-DBUILD_ASSEMBLY=OFF
+          #-DBUILD_CLOUD=OFF
+          #-DBUILD_COMPLETE=OFF
+          #-DBUILD_DRAFT=OFF
+          #-DBUILD_DRAWING=OFF
+          #-DBUILD_FEM=OFF
+          #-DBUILD_FEM_NETGEN=OFF
+          #-DBUILD_GUI=OFF
+          #-DBUILD_IDF=OFF
+          #-DBUILD_IMAGE=OFF
+          #-DBUILD_IMPORT=OFF
+          #-DBUILD_INSPECTION=OFF
+          #-DBUILD_JTREADER=OFF
+          #-DBUILD_MATERIAL=OFF
+          #-DBUILD_MESH=OFF
+          #-DBUILD_MESH_PART=OFF
+          #-DBUILD_OPENSCAD=OFF
+          #-DBUILD_PART=ON
+          #-DBUILD_PART_DESIGN=OFF
+          #-DBUILD_PATH=OFF
+          #-DBUILD_PLOT=OFF
+          #-DBUILD_POINTS=OFF
+          #-DBUILD_RAYTRACING=OFF
+          #-DBUILD_REVERSEENGINEERING=OFF
+          #-DBUILD_ROBOT=OFF
+          #-DBUILD_SANDBOX=OFF
+          #-DBUILD_SHOW=OFF
+          #-DBUILD_SKETCHER=OFF
+          #-DBUILD_SPREADSHEET=OFF
+          #-DBUILD_START=OFF
+          #-DBUILD_SURFACE=OFF
+          #-DBUILD_TECHDRAW=OFF
+          #-DBUILD_TEMPLATE=OFF
+          #-DBUILD_TEST=ON
+          #-DBUILD_TUX=OFF
+          #-DBUILD_WEB=OFF
+##
+      - name: Compiling sources
+## NINJA
+        #run: cd $CI_BUILD_DIR && ninja -j$(nproc)
+## CCACHE
+        run: cd $CI_BUILD_DIR && cmake --build ./ -j$(nproc)
+##
+      - name: Installing
+        run: cd $CI_BUILD_DIR && cmake --install ./
+      - name: (DEBUG) Search executables
+        run: find / -type f -iname "*freecadcmd*"
+      - name: Print ccache statistics
+        run: ccache -s
+      - name: Assess compiled version
+        run: $CI_BUILD_DIR/bin/FreeCADCmd --version
+      - name: Testing
+        run: xvfb-run $CI_BUILD_DIR/bin/FreeCAD -t 0 # Test in GUI mode
```

Comment:
I'm a little confused here, I think this is still running the test on the binary in the build directory, I think we want to run it in the installed binary

Reply:
Huhhh. Don't expect me to do it in a blink of an eye. I'm currently running a dry-run to check where "install" will place the binaries. Then I'll change this line. I'll post here when it's over. ;)

Reply:
Thanks, sorry for the noise